pkg/services/ngalert/notifier/alertmanager_config.go

```diff
@@ -0,0 +1,145 @@
+package notifier
```

Comment:
I'm sorry, I could have been clearer with my suggestions. I don't think this matches what I originally suggested.

I meant to have `GetAlertmanagerConfiguration` and `ApplyAlertmanagerConfiguration` be part of `*multiOrg` even if its split in a different file and uses a separate interface.

Reply:
Ah - I understand! Yeah, that is pretty slick, and a lot simpler. The original change only made it halfway there - correct package, but not attached to the same type itself. `multiOrg` is definitely a very relevant object to attach these to.